src/CodeGenerator/Generator/RequestSerializer/QuerySerializer.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare(strict_types=1);
+
+namespace AsyncAws\CodeGenerator\Generator\RequestSerializer;
+
+use AsyncAws\CodeGenerator\Definition\ListShape;
+use AsyncAws\CodeGenerator\Definition\MapShape;
+use AsyncAws\CodeGenerator\Definition\Member;
+use AsyncAws\CodeGenerator\Definition\Operation;
+use AsyncAws\CodeGenerator\Definition\Shape;
+use AsyncAws\CodeGenerator\Definition\StructureMember;
+use AsyncAws\CodeGenerator\Definition\StructureShape;
+use AsyncAws\CodeGenerator\Generator\RequestSerializer\Serializer;
+
+/**
+ * @author Jérémy Derussé <jeremy@derusse.com>
+ *
+ * @internal
+ */
+class QuerySerializer extends ArraySerializer
+{
+
+    public function getContentType(): string
+    {
+        return 'application/x-www-form-urlencoded';
+    }
+
+    protected function convertFromArray(string $variableName)
+    {
+        return <<<PHP
+// Official SQK: return http_build_query($variableName, null, '&', \PHP_QUERY_RFC3986);
```

Comment:
SQK?

this comment should not be dumped IMHO

Reply:
Agreed, It is only when WIP. 

I'll leave this review comment open

Reply:
I dont dump this comment anymore